dhis-2/dhis-api/src/main/java/org/hisp/dhis/trackedentity/TrackedEntityInstanceQueryParams.java

```diff
@@ -223,6 +223,11 @@
      * Indicates whether paging should be skipped.
      */
     private boolean skipPaging;
+    
+    /**
+     * Indicates if there is a maximum tei retrieval limit. 0 or null means no limit.
```

Comment:
This is a primitive int, it cannot be null

Reply:
Good catch! It was a wrapper Integer initially and changed after a while and missed out on changing the comment.